src/docs/_data/navigation.yml

```diff
@@ -0,0 +1,67 @@
+- name: Basics
```

Comment:
Create component in `/components` folder and mention it in this file - everything else happens automatically.